src/style.css

```diff
@@ -1,3 +1,95 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --select-color: #00acdc;
+  --main-color: #000;
+  --font-family: roboto, sans-serif;
+  --main-text-size: 16px;
+}
+
+.body {
   margin: 0;
+  font-family: var(--font-family);
+  font-size: var(--main-text-size);
+  color: var(--main-color);
+  line-height: normal;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+.nav-list__link {
+  align-items: center;
+  display: flex;
+  height: 100%;
+}
+
+.header__logo__image {
+  width: 40px;
+  height: 40px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-list__item {
+  list-style-type: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  align-items: center;
+  height: 100%;
+  display: flex;
+}
+
+.nav-list__link:hover {
+  color: var(--select-color);
+}
+
+.nav-list__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav-list__link.is-active {
+  position: relative;
+  color: var(--select-color);
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  bottom: 0;
+  left: 0;
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background-color: var(--select-color);
+}
+
+.header {
+  align-items: center;
+  height: 60px;
```

Comment:
Per checklist:
`Don't set height for header explicitly. Let the content (links) dictate it.`

Hint: if you will set height (explicitly or by padding) for one of the inner elements, you will not need those ugly `height: 100%` in so many places